### PR TITLE
[FTAMS102020] Ahmet Demirlicakmak

### PR DESCRIPTION
**Pull request by codebrokerad**
Done until iteration 4

### CODE REVIEW

File: src/components/Randombeer/Randombeer.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+const BeerDetails = (props) => {
+  const [beerDetails, setBeerDetails] = useState({});
+
+  useEffect(() => {
+    const { id } = props.match.params;
+    axios.get(`https://ih-beers-api2.herokuapp.com/beers/${id}`).then((res) => {
+      const beerDetails = res.data;
+      setBeerDetails(beerDetails);
+    });
+  });
```
Comment (LloydC) [inline src/components/Randombeer/Randombeer.js:14]: You forgot to add a dependency array here.
```
  }, []);

File: src/components/Beers/Beers.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+export default class Beers extends React.Component {
+  state = {
+    beers: [],
+  };
+
+  componentDidMount() {
+    axios.get(`https://ih-beers-api2.herokuapp.com/beers`).then((res) => {
+      const beers = res.data;
+      this.setState({ beers });
+    });
```
Comment (LloydC) [inline src/components/Beers/Beers.js:16]: You forgot to add a `.catch(err => console.log(err))`.

File: src/components/Beers/BeerDetails.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+const BeerDetails = (props) => {
+  const [beerDetails, setBeerDetails] = useState({});
+
+  useEffect(() => {
+    const { id } = props.match.params;
+    axios.get(`https://ih-beers-api2.herokuapp.com/beers/${id}`).then((res) => {
+      const beerDetails = res.data;
+      setBeerDetails(beerDetails);
+    });
```
Comment (LloydC) [inline src/components/Beers/BeerDetails.js:13]: You forgot to add a .catch(err => console.log(err)).

File: src/components/Beers/BeerDetails.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  },);
```
Comment (LloydC) [inline src/components/Beers/BeerDetails.js:14]: You should have a dependency array here as well.
```
}, []);

File: src/components/Beers/Beers.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  }
+
+  render() {
+    return (
+      <ul>
+        <Navbar />
+        {this.state.beers.map((beers) => (
+          <li>
```
Comment (LloydC) [inline src/components/Beers/Beers.js:24]: You forgot to add the `key` attribute.
```
<li key={beers._id}>

File: src/components/Beers/BeerDetails.js (hunk 4 above)
Comment (codebrokerad) [reply src/components/Beers/BeerDetails.js:14]: It was there, but giving an error on the terminal although compiling the page well, thus I removed it. I can add again.

**Pull request closed without merging**